io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -93,6 +96,44 @@ public:
     */
    void SetMergeOptions(const TString& options);
 
+   /** Indicates that the file will not contains any TTree objects
+    * and thus that steps that are specific to TTree can be skipped */
+   void SetNotrees(Bool_t notrees=kFALSE)
```

Comment:
In old ROOT style this should be `SetNoTrees`

Reply:
Actually this is a copy/paste of the function name it forward to.  I.e. we made a typo in 2007 in TFileMerger and for backward compatibility I choose to keep it that way in TFileMerger and since this is (essentially) a forwarding function, to use the same spelling here.